redis_enterprise/metadata.csv

```diff
@@ -1,180 +1,180 @@
-rdse.redis_process_main_thread_cpu_user_seconds_total,gauge,15,second,,Shard main thread user CPU time spent in seconds,1,redis_enterprise_shard,redis_process_main_thread_cpu_user_seconds_total,
-rdse.redis_process_max_fds,gauge,15,file,,Shard Maximum number of open file descriptors,1,redis_enterprise_shard,redis_process_max_fds,
-rdse.redis_process_open_fds,gauge,15,file,,Shard Number of open file descriptors,0,redis_enterprise_shard,redis_process_open_fds,
-rdse.redis_process_resident_memory_bytes,gauge,15,byte,,Shard Resident memory size in bytes,0,redis_enterprise_shard,redis_process_resident_memory_bytes,
-rdse.redis_process_start_time_seconds,gauge,15,second,,Shard Start time of the process since unix epoch in seconds,1,redis_enterprise_shard,redis_process_start_time_seconds,
-rdse.redis_process_virtual_memory_bytes,gauge,15,byte,,Shard virtual memory in bytes,0,redis_enterprise_shard,redis_process_virtual_memory_bytes,
-rdse.redis_rdb_bgsave_in_progress,gauge,15,check,,Indication if bgsave is currently in progress,0,redis_enterprise_shard,redis_rdb_bgsave_in_progress,
-rdse.redis_rdb_last_cow_size,gauge,15,byte,,Last bgsave (or SYNC fork) used CopyOnWrite memory,0,redis_enterprise_shard,redis_rdb_last_cow_size,
-rdse.redis_rdb_saves,gauge,15,operation,,Total count of bgsaves since process was restarted (including replica fullsync and persistence),1,redis_enterprise_shard,redis_rdb_saves,
-rdse.redis_repl_touch_bytes,gauge,15,byte,,Number of bytes sent to replicas as TOUCH commands by the shard as a result of a READ command that was processe,0,redis_enterprise_shard,redis_repl_touch_bytes,
-rdse.redis_total_commands_processed,gauge,15,command,,Number of commands processed by the shard; Calculate the number of commands for a time period by comparing the value at different times,1,redis_enterprise_shard,redis_total_commands_processed,
-rdse.redis_total_connections_received,gauge,15,connection,,Number of connections received by the shard; Calculate the number of connections for a time period by comparing the value at different times,1,redis_enterprise_shard,redis_total_connections_received,
-rdse.redis_total_net_input_bytes,gauge,15,request,,Number of bytes received by the shard; Calculate the throughput for a time period by comparing the value at different times,1,redis_enterprise_shard,redis_total_net_input_bytes,
-rdse.redis_total_net_output_bytes,gauge,15,response,,Number of bytes sent by the shard; Calculate the throughput for a time period by comparing the value at different times,1,redis_enterprise_shard,redis_total_net_output_bytes,
-rdse.redis_up,gauge,15,service,,Shard is up and running,0,redis_enterprise_shard,redis_up,
-rdse.redis_used_memory,gauge,15,byte,,Memory used by shard (in bigredis this includes flash) (bytes),0,redis_enterprise_shard,redis_used_memory,
+rdse.bdb_avg_latency,gauge,15,second,,Average latency of operations on the DB (seconds); returned only when there is traffic,-1,redis_enterprise_v2,bdb_avg_latency,
```

Comment:
@j8-redis out of curiosity, where does this field show up that you need to change it?

Reply:
that's a typo or fat-finger, it has been removed